src/main/resources/org/jboss/xavier/analytics/rules/workload/inventory/Workloads.drl

```diff
@@ -290,3 +290,48 @@ rule "SsaDisabled_System_Services_Not_Present"
         }
 
 end
+
+rule "Workloads_Oracle_JDK_8_On_Linux"
+       when
+           vmWorkloadInventoryModel : VMWorkloadInventoryModel(
+               files != null,
+               files.size() > 0,
+               fileNameAndContentsContainSpecifiedStrings(this, "/usr/java/latest/release","JAVA_VERSION=\\\"1.8")
```

Comment:
The folder would not depend on the different Jdk installed ? I mean that machine could have Java 8 and Java 11 , and then latest would not be 8 but 11